### PR TITLE
Update dependencies.props

### PR DESCRIPTION
**Pull request by sebastienros**
~~I am still waiting for EF packages to be on myget before updating their versions to the very latest, but the build passed.~~

### CODE REVIEW

File: build/dependencies.props [hunk 1]
```diff
@@ -236,7 +236,7 @@
     <SystemIdentityModelTokensJwtPackageVersion>5.3.0</SystemIdentityModelTokensJwtPackageVersion>
     <SystemNetHttpPackageVersion>4.3.2</SystemNetHttpPackageVersion>
     <SystemReactiveLinqPackageVersion>3.1.1</SystemReactiveLinqPackageVersion>
-    <SystemReflectionEmitPackageVersion>4.3.0</SystemReflectionEmitPackageVersion>
+    <SystemReflectionEmitPackageVersion>4.6.0-preview.18606.1</SystemReflectionEmitPackageVersion>
```
Comment (sebastienros) [inline build/dependencies.props]: Is that something that needs to be added to https://github.com/dotnet/versions/pull/409 ?

File: build/dependencies.props [hunk 2]
```diff
@@ -191,7 +191,7 @@
     <MicrosoftNETCoreDotNetAppHost22PackageVersion>$(MicrosoftNETCoreApp22PackageVersion)</MicrosoftNETCoreDotNetAppHost22PackageVersion>
     <MicrosoftNETCoreWindowsApiSetsPackageVersion>1.0.1</MicrosoftNETCoreWindowsApiSetsPackageVersion>
     <MicrosoftNETFrameworkReferenceAssembliesPackageVersion>1.0.0-alpha-004</MicrosoftNETFrameworkReferenceAssembliesPackageVersion>
-    <MicrosoftNETTestSdkPackageVersion>15.6.1</MicrosoftNETTestSdkPackageVersion>
+    <MicrosoftNETTestSdkPackageVersion>15.9.0</MicrosoftNETTestSdkPackageVersion>
```
Comment (sebastienros) [inline build/dependencies.props]: This one is coming from EF dependencies.props

File: build/dependencies.props (hunk 1 above)
Comment (natemcmaster) [reply build/dependencies.props]: This looks right to me.

Comment (natemcmaster) [reply build/dependencies.props]: Can you move this version up near the other files that updated (near the comment "Packages from dotnet/corefx ")?

**Comment by natemcmaster** [comment]
PR checks are failing because of this:
```
==== unexpected assemblies in the manifest ====
Microsoft.Win32.Registry
System.Diagnostics.DiagnosticSource
System.Reflection.Metadata
System.Security.AccessControl
System.Security.Cryptography.Cng
System.Security.Principal.Windows
```
This happens when either
1. the corefx package versions are not coherent with the version of MicrosoftNETCoreAppPackageVersion. If the corefx package versions are newer, we'll end up with extra assemblies in our shared framework
2. netcoreapp removed something from the shared framework

It's usually (1). You can verify by looking at https://github.com/dotnet/core-setup/blob/master/dependencies.props to see if core-setup has built with new corefx packages yet.

**Comment by sebastienros** [comment]
This is my change:

```
    <MicrosoftNETCoreAppPackageVersion>3.0.0-preview-27205-02</MicrosoftNETCoreAppPackageVersion>
    <MicrosoftNETCoreDotNetAppHostPackageVersion>3.0.0-preview-27206-02</MicrosoftNETCoreDotNetAppHostPackageVersion>
    <MicrosoftDotNetPlatformAbstractionsPackageVersion>3.0.0-preview-27206-02</MicrosoftDotNetPlatformAbstractionsPackageVersion>
```

I can see two issue:
- I have mixed versions, `27205-02` and `27206-02`, probably because I got `MicrosoftNETCoreAppPackageVersion` a day before the other two in Extensions, and floweed the same value for consistency for EF and AspNetCore
- The file you pointed me to has `3.0.0-preview.18606.1` and `3.0.0-preview-27206-02`

If I want to update to `3.0.0-preview-27206-02` should it start again from Extensions? Or can I just change it here?

**Comment by natemcmaster** [comment]
It looks like we need a new build of core-setup before we can take corefx 18606.1.

https://github.com/dotnet/core-setup/compare/a1e4845188a38880143164aa4da6a480eed8e746...master

@wtgodbe @vitek-karas - it looks like the changes were merged to update core-setup, but we're not seeing a new version of Microsoft.NETCore.App on public feeds. Are there known issues building core-setup?

**Comment by vitek-karas** [comment]
@rakeshsinghranchi

**Comment by sebastienros** [comment]
Could I try with a previous version until someone looks into it?

Like:
https://github.com/dotnet/core-setup/commit/c6c782a98cb7c61746a08a9574ee5b2c3b77894e#diff-54339bb49aae3cad78638955e8cda9f7

Still better than 2 weeks old.

**Comment by natemcmaster** [comment]
@sebastienros we could have to revert Extensions (https://github.com/aspnet/Extensions/pull/591) and EFCore to older versions of corefx that have a matching core-setup build.

**Comment by sebastienros** [comment]
That's what I am suggesting. Update extensions, ef, and aspnetcore to these older versions.

**Comment by natemcmaster** [comment]
That's up to you. I don't know which will go faster -- rebuilding from extensions-upward with older corefx dependencies or waiting for a new build of core-setup.

**Comment by natemcmaster** [comment]
@sebastienros if you're still blocked on this, can you send email internally and cc @Eilon? We probably need to elevate to our partner teams.

**Comment by rakeshsinghranchi** [comment]
There is a build in progress from core-setup master branch.

**Comment by sebastienros** [comment]
If I look at .NET Mission Control, I can see 10 builds (3 for today) on core-setup. So when you say "There is a build in progress", do you actually mean the issue has been found and the next build is expected to be green?

**Comment by rakeshsinghranchi** [comment]
Core-Setup build issue has been resolved now. Today's  build is available . https://github.com/dotnet/versions/blob/master/build-info/dotnet/core-setup/master/Latest_Packages.txt

**Pull request closed without merging**